src/handlers/testing/align/util/session-manager.ts

```diff
@@ -0,0 +1,134 @@
+import crypto from 'crypto';
+import { writeEvaluatorTemplates } from './autogenerate';
+import { EventName, emitter } from './emitter';
+import {
+  Language,
+  type AutogeneratedEvaluator,
+  type TestCaseEvent,
+  type TestCaseResult,
+  type GradedTestCaseResult,
+} from './models';
+
+export class SessionManager {
+  private readonly apiKey: string;
+  private readonly testExternalId: string;
+  private sessionId: string | undefined;
+  private language: Language | undefined = Language.PYTHON;
+  private runTestSuiteCalledFromFilepath: string | undefined =
+    '/Users/nicole/autoblocks/cli';
```

Comment:
Should this be an env var? 

Assuming this is just temporary until we hook everything up?

Reply:
this was just for testing, i'll remove